### DISCUSSION TITLE
math code block syntax is broken

### DISCUSSION THREAD

**Discussion by lizelive** [community/community#28975] [Other Feature Feedback, Questions, & Ideas]
[the docs ](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions#writing-expressions-as-blocks) say that 

**Here is some math!**

```math
\sqrt{3}
```

should result in ![math-expression-as-a-fenced-code-block](https://user-images.githubusercontent.com/40217331/183827573-13b7d6d0-4f7e-481e-9b6a-e5b55e7d116c.png)

but it currently just shows as 

```
\sqrt{3}
```

**Comment by nschloe** [comment]
_Edit Nov 21, 2022:_ This seems to have been fixed now.

This syntax is only enabled on old repos, and some new ones. Not this one yet, it seems.

Test:
````markdown
```math
\sqrt{3}
```
````

```math
\sqrt{3}
```

**Comment by mh105** [comment]
Has there been a resolution of this issue? I just tested. Seems like the [fenced block syntax](https://github.blog/changelog/2022-06-28-fenced-block-syntax-for-mathematical-expressions/) is only enabled for private repositories but not for public ones.

**Reply by nschloe** [reply]
It's not private vs public, it's only enabled for _new_ repos. Old ones don't work with fenced syntax, see https://github.com/orgs/community/discussions/20934.

**Reply by mh105** [reply]
> It's not private vs public, it's only enabled for _new_ repos.

Nope. I just created a new public repo - does not work. Could you try it as well and see if it works for you on a new public repo?

**Reply by nschloe** [reply]
Aha, interesting. I might try later.

**Reply by mh105** [reply]
> Aha, interesting. I might try later.

Yeah and also try turning it into a private afterwards. You'll see fenced math suddenly works when you turn it into a private repo. Turning back to public again disabled it.

**Reply by mh105** [reply]
FYI: https://github.com/github/docs/issues/21569#issuecomment-1292420499

**Reply by therzka** [reply]
We've fixed the issue where this syntax was only working for private repos :)

**Reply by nschloe** [reply]
@therzka The original question above still shows

![screenshot](https://user-images.githubusercontent.com/181628/202203287-915d338a-5def-4023-a918-a0533f80c931.png)

Can you confirm this?

**Reply by mh105** [reply]
> Can you confirm this?

Seems to work for me in repos but not here in discussion threads.

**Reply by therzka** [reply]
looking into it!

**Reply by nschloe** [reply]
@therzka Apparently it works now!

**Comment by Yury-Fridlyand** [comment]
On my repo I see `math` block is rendered properly if there is nothing else in the comment. Meantime, preview doesn't work at all.
